roles/deploy-host/tasks/main.yaml

```diff
@@ -39,16 +39,17 @@
       command: "subscription-manager repos --enable=rhel-7-server-rhv-4-mgmt-agent-rpms"
       when: "'rhv' not in enabled_repos.stdout"
 
+    - name: "Enable oVirt repository"
+      command: 'yum -y install http://resources.ovirt.org/pub/yum-repo/ovirt-release42.rpm'
+
```

Comment:
Shouldn't this be enabled ('when' statement) only on oVirt and not for RHV?

Reply:
This part is just to access the ovirt-ansible rpm. I didn't see it available in the rhel-7-server-rhv-4-mgmt-rpms repo (yet).

Reply:
https://access.redhat.com/errata/product/150/ver=4.1/rhel---7/x86_64/RHBA-2017:3137